lang/attribute/state/src/lib.rs

```diff
@@ -45,7 +45,7 @@ pub fn state(
                     fn size(&self) -> anchor_lang::Result<u64> {
                         Ok(8 + self
                             .try_to_vec()
-                            .map_err(|_| anchor_lang::error::Error::from(anchor_lang::error::ErrorCode::AccountDidNotSerialize))?
```

Comment:
I'm expecting all these conversions to happen implicitly. We'll see if the tests pass :shrug:

Reply:
ok haha I just thought of the same and was about to adjust it. yes it works like this. My brain just stopped working for a bit. Ill do the rest

Reply:
ah you did everything already. nice

Reply:
i think i missed a couple...wasn't sure if it would work in the first place

Reply:
i think we got em all now